Makefile

```diff
@@ -0,0 +1,15 @@
+SHELL := /bin/bash
+
+help:
+	@echo "usage:"
+	@echo "    make docs    -- setup and serve docs locally"
+	@echo "    make release -- build and release to PyPI"
+
+docs:
+	pip install -r doc/requirements.txt
+	mkdocs serve
+
+release:
+	rm -rf dist/*
+	python setup.py register bdist_wheel sdist
```

Comment:
Why register here ? I thought you only need to do that once ?

Reply:
Register sets metadata on the PyPI so if you ever change something like Trove classifiers or author email in your setup.py it wouldn't be updated in the PyPI until you run `register` again.

Doing it on every release is probably overkill but it removes you ever having to think about it and is harmless when nothing has changed.

Reply:
I did not know that upload wouldn't update metadata !